src/components/Developers/Developers.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import {Row,Container,Col} from 'react-materialize'
+import './Developers.css'
+
+class Developers extends Component {
+	render() {
+		//Const or fetch API
+		const users=[
+			{id:0,name:"User",img:"https://ui-avatars.com/api/?size=216&name=User",link:[{name:"whatsapp",url:""},{name:"facebook",url:""},{name:"twitter",url:""}]},
+			{id:1,name:"User",img:"https://ui-avatars.com/api/?size=216&name=User",link:[{name:"whatsapp",url:""},{name:"linkedin",url:""},{name:"snapchat",url:""}]},
+			{id:2,name:"User",img:"https://ui-avatars.com/api/?size=216&name=User",link:[{name:"whatsapp",url:""},{name:"instagram",url:""},{name:"quora",url:""},{name:"github",url:""}]},
+		];
+		
+		//For linear Gradiant
+		const colors=[
+			{init:"D21DB0",final:"9133BF"},
+			{init:"FFB707",final:"FF6401"},
+			{init:"00FDFD",final:"00C6FC"}
+		]
+		return (
+			<React.Fragment>
+				<center>	
+					<h1>Our Team</h1>
+					</center>
+				<Container>
+<Row>
```

Comment:
@ankurcharan is this the right indentation?
Maybe @arpitBhalla can put this code in some other file and then populate it from there. It might look cleaner?